### DISCUSSION TITLE
Support for fallback hubUrls

### DISCUSSION THREAD

**Discussion by ggomaeng** [wevm/frog-archived#158] [Ideas]
It would be pretty neat to pass in an array of hubUrls like Viem's [fallback](https://viem.sh/docs/clients/transports/fallback).

There are times when the hub is down and it just makes the app non-functional.

Current 

```ts
import { Frog } from 'frog'
 
const app = new Frog({
  hub: {
    apiUrl: 'https://api.hub.wevm.dev'
  }
})
```

Proposal
```ts
import { Frog } from 'frog'
 
const app = new Frog({
  hub: {
    apiUrl: ['https://api.hub.wevm.dev', 'https://freefarcasterhub.com', ...]
  }
})
```

I used to do something like below before finding frog 🐸

https://gist.github.com/ggomaeng/7895a2e410874ca87f5534890055d20c

**Comment by dalechyn** [comment]
Moved to issue https://github.com/wevm/frog/issues/488
